concent_api/conductor/tests/test_integration_verification.py

```diff
@@ -126,17 +130,81 @@ def test_conductor_should_schedule_verification_order_task_if_all_related_upload
-        self.assertEqual(upload_report.path, self.file_path)
+        self.assertEqual(upload_report.path, self.source_package_path)
+
+        mock_task.assert_called()
+
+    def test_blender_verification_request_task_should_create_verification_request_and_blender_subtask_definition(self):
+        blender_verification_request(
+            subtask_id=self.compute_task_def['subtask_id'],
+            source_package_path=self.source_package_path,
+            result_package_path=self.result_package_path,
+            output_format=BlenderSubtaskDefinition.OutputFormat.JPG.name,  # pylint: disable=no-member
+            scene_file=self.scene_file,
+        )
+
+        self.assertEqual(VerificationRequest.objects.count(), 1)
+
+        verification_request = VerificationRequest.objects.first()
+        self.assertEqual(verification_request.subtask_id,  self.compute_task_def['subtask_id'])
+        self.assertEqual(verification_request.source_package_path, self.source_package_path)
+        self.assertEqual(verification_request.result_package_path, self.result_package_path)
+        self.assertEqual(verification_request.blender_subtask_definition.output_format, BlenderSubtaskDefinition.OutputFormat.JPG.name)  # pylint: disable=no-member
+        self.assertEqual(verification_request.blender_subtask_definition.scene_file, self.scene_file)
+
+    def test_blender_verification_request_task_should_not_link_upload_requests_to_unrelated_upload_reports(self):
+        upload_report = UploadReport(
+            path='blender/scene/bad/bad.bad.zip',
+        )
+        upload_report.full_clean()
+        upload_report.save()
+
+        blender_verification_request(
+            subtask_id=self.compute_task_def['subtask_id'],
+            source_package_path=self.source_package_path,
+            result_package_path=self.result_package_path,
+            output_format=BlenderSubtaskDefinition.OutputFormat.JPG.name,  # pylint: disable=no-member
+            scene_file=self.scene_file,
+        )
+
+        self.assertEqual(VerificationRequest.objects.count(), 1)
+
+        verification_request = VerificationRequest.objects.first()
+        self.assertEqual(verification_request.upload_reports.count(), 0)
+        self.assertFalse(verification_request.upload_reports.filter(path=self.source_package_path).exists())
+
+    @override_settings(CELERY_TASK_ALWAYS_EAGER=True)
+    def test_blender_verification_request_task_should_schedule_verification_order_task_if_all_related_upload_requests_have_reports(self):
+        upload_report = UploadReport(
+            path=self.source_package_path,
+        )
+        upload_report.full_clean()
+        upload_report.save()
+
+        upload_report = UploadReport(
+            path=self.result_package_path,
+        )
+        upload_report.full_clean()
+        upload_report.save()
+
+        with mock.patch('conductor.tasks.blender_verification_request.delay') as mock_task:
```

Comment:
you could use `mock_task.assert_called_once_with(*args)` to verify proper call of the mock

Reply:
:+1: done